src/ui/server/test-mocks/mock-eligibility.ts

```diff
@@ -19,6 +19,8 @@ const mockEligibility = {
   totalContractValue: TOTAL_CONTRACT_VALUE.LESS_THAN_500K.DB_ID,
   company,
   hasReviewedEligibility: true,
+  partyToConsortium: false,
+  memberOfAGroup: false,
```

Comment:
Just to confirm is this a happy path? I.e answering "no" to both questions, a user isn't redirected to any exit pages

Reply:
correct